src/core/positioning/serialportreceiver.h

```diff
@@ -41,6 +44,8 @@ class SerialPortReceiver : public NmeaGnssReceiver
     QString mAddress;
 
     QSerialPort *mSocket = nullptr;
```

Comment:
Its a bit strange to call the serial port "Socket". This as well doesn't need to be a pointer

```suggestion
    QSerialPort *mSerialPort = nullptr;
```